### PR TITLE
fix: replit issues

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

related to https://github.com/freeCodeCamp/freeCodeCamp/issues/46910

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Review by lasjorg — APPROVED** [review]
LGTM

File: .replit [hunk 1]
```diff
@@ -1,2 +1,10 @@
+run = "npm start"
+entrypoint = "server.js"
+
+[packager]
 language = "nodejs"
-run = "npm start"
\ No newline at end of file
+
+  [packager.features]
```
Comment (ShaunSHamilton) [inline .replit:7]: Do we need this spacing?

File: .replit [hunk 2]
```diff
@@ -1,2 +1,10 @@
+run = "npm start"
+entrypoint = "server.js"
+
+[packager]
 language = "nodejs"
```
Comment (ShaunSHamilton) [inline .replit:5]: What is the point of having this, if we still need to add that line to `.gitattributes`?

**Review by ShaunSHamilton — APPROVED** [review]
I am happy with this, but have questions out of interest.

File: .replit (hunk 1 above)
Comment (lasjorg) [reply .replit:7]: It just how it is formatted in the Replit docs.

File: .replit (hunk 2 above)
Comment (lasjorg) [reply .replit:5]: https://docs.replit.com/programming-ide/configuring-repl#packager

> [packager]
> Description: Package management configuration. Learn more about installing packages [here](https://docs.replit.com/repls/packages/#DirectImports).

That config setting doesn't seem to affect the language auto-detection Replit does when importing from GitHub. As long as Linguist detects some language as the one with the most code (i.e. the repo language statistics) Replit seems to use that as the information about which language it should auto-detect.

**Pull request merged**